app/root.tsx

```diff
@@ -3,20 +3,59 @@ import type {MetaFunction, LinksFunction, LoaderFunction} from '@remix-run/cloud
 import styles from '~/root.css'
 
 import {useLoaderData} from '@remix-run/react'
+import {getStateEntries} from '~/hooks/stateModifiers'
+import {loadQuestionDetail} from '~/server-utils/stampy'
 
-const title = 'Stampy'
-const description = 'AI Safety FAQ'
+/*
+ * Transform the given text into a meta header format.
+ *
+ * In practice, this means stripping out any HTML tags and limiting its length
+ */
+const metafyText = (text: string | null, defaultText: string) => {
+  if (!text || text.length == 0) return defaultText
+
+  let cleaned = text
+  // Totally remove any P, A or IFRAME tags, but leave the contents. Any other
+  // tags will be ignored for now
+  const removals = [/<\/?p.*?>/gi, /<\/?a.*?>/gi, /<\/?iframe.*?>/gi]
+  cleaned = removals.reduce((str, regex) => str.replace(regex, ''), cleaned)
+
+  // This first random website I found claims that the description limit is 155,
+  // so keep it to that: https://www.contentkingapp.com/academy/meta-description/
+  if (cleaned.length > 155) cleaned = cleaned.slice(0, 152) + '...'
```

Comment:
have you tested on Discord, Slack, Twitter, and Facebook yet? (what else do we care about?) we should aim for shortest supported by all (155 if fine if none of the sites shrinks it even more - I just don't want to force editors to test it on multiple sites for "unfortunate" shortening variations)

Reply:
Discord and Slack work, I don't have Twitter or Facebook, so can't check them

Reply:
twitter doesn't work (probably never worked in previous version either, not a new bug, but worth fixing nevertheless):

![image](https://user-images.githubusercontent.com/1087670/221417900-530d18af-be57-41d5-bc47-58f9c02e75b5.png)

error message on https://cards-dev.twitter.com/validator `ERROR: No card found (Card error)` does not sound very useful, but feel free to use the validator for any fixing ideas

linkedin doesn't like it either:
![image](https://user-images.githubusercontent.com/1087670/221418274-8354d46c-97bb-4674-87a1-c523364016f7.png)

facebook looks half-way there:
![image](https://user-images.githubusercontent.com/1087670/221418119-be59f44e-540e-45ff-b7bf-0cca0e9468c2.png)

Reply:
even on Discord, the image is missing (which is probably why you didn't spot that it's a wrong image)

Reply:
so, @plexish wants to triple the text in Discord (discussed in https://discord.com/channels/677546901339504640/758062805810282526/1079418708394512455) => feel free to increase the limit even if it would get shortened on other platforms, not a big deal for me to keep the text length consistent